Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM adoptopenjdk/openjdk11:jre
```

Comment:
Adoptopenjdk seems like depreciated.
https://github.com/AdoptOpenJDK/openjdk-docker#-deprecation-notice-

Also looks like the AdoptOpenJDK moved to [Eclipse Adoptium](https://projects.eclipse.org/projects/adoptium):
 https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/

May be explore a bit on this and completely move out of adoptopenjdk?
 
Examples of some PR's to migrate to Eclipse temurin in this issue:
https://github.com/docker-library/openjdk/issues/505


If you think, we need Evaluation or needs more time to understand, please create a ticket.

Reply:
Good catch 🐱 However, it is not in scope of my task. Thus I created follow-up ticket for this: https://jira.commercetools.com/browse/CTPI-698